vulnerability-analyzer/src/test/java/org/dependencytrack/vulnanalyzer/client/snyk/SnykClientTest.java

```diff
@@ -150,33 +143,21 @@ void testGetIssuesUsingBatchApiWillReturn500WhenPurlInBatchExceedLimit() throws
         );
 
         // Verify that we sent the request in the correct format
-        wireMockServer.verify(postRequestedFor(urlPathMatching("/rest/orgs/test-org-id/packages/issues"))
+        wireMock.verifyThat(postRequestedFor(urlPathMatching("/rest/orgs/test-org-id/packages/issues"))
                 .withQueryParam("version", matching("\\d{4}-\\d{2}-\\d{2}~experimental"))
                 .withHeader(HttpHeaders.AUTHORIZATION, matching("token token-\\d"))
                 .withHeader(HttpHeaders.ACCEPT, equalTo("application/vnd.api+json"))
                 .withHeader(HttpHeaders.USER_AGENT, containing(System.getProperty("os.arch") + "; ")));
 
         assertThat(thrown)
                 .hasMessageContaining("requestId=snykRequestId666")
-                .hasMessageContaining("""
-                        {
-                          "jsonapi": {
-                            "version": "1.0"
-                          },
-                          "errors": [
-                            {
-                              "id": "0d7d9f37-b729-4dab-8356-1fd22520fdcc",
-                              "detail": "An unknown error was encountered while handling your request. Please contact support.",
-                              "status": "500"
-                            }
-                          ]
-                        }""");
+                .hasMessageContaining("{\"jsonapi\":{\"version\":\"1.0\"},\"errors\":[{\"id\":\"0d7d9f37-b729-4dab-8356-1fd22520fdcc\",\"detail\":\"An unknown error was encountered while handling your request. Please contact support.\",\"status\":\"500\"}]}");
```

Comment:
Any reason for this change? Makes it a lot harder to read I think.

Reply:
Yes it is failing for exactly same Json if extended/formatted like before, I'm not sure why.